src/components/LibraryContainer.tsx

```diff
@@ -194,6 +194,7 @@ export class LibraryContainer extends React.Component<LibraryContainerProps, Lib
 
     render() {
         if (!this.generatedSections) {
+            this.props.libraryController.request(this.props.libraryController.RefreshLibraryViewRequestName, function() {});
```

Comment:
The function should be `null` if we change the implementation of `request` method as suggested towards the end. And I personally don't like lengthy lines of codes 😄 

```js
let eventName = this.props.libraryController.RefreshLibraryViewRequestName;
this.props.libraryController.request(eventName, null);
```